package.json

```diff
@@ -25,5 +26,8 @@
     "test": "npm-run-all check-formatting build wdio-test",
     "watch": "npm run build -- --watch",
     "wdio-test": "npx wdio run wdio.conf.js"
+  },
+  "dependencies": {
+    "postcss": "^8.4.8"
```

Comment:
No need to explicitly include `postcss`. It's only used by Tailwind and if they stop using it in the future, you'd continue installing it because you have it listed as your dependency.